stdlib/public/core/UnsafePointer.swift.gyb

```diff
@@ -535,9 +541,8 @@ public struct ${Self}<Pointee>
   ///   - count: The number of instances to copy from the memory referenced by
   ///     `source` to this pointer's memory. `count` must not be negative.
   @_inlineable
-  public func assign(from source: UnsafePointer<Pointee>, count: Int) {
-    _debugPrecondition(
-      count >= 0, "${Self}.assign with negative count")
+  public func assign(from source: UnsafePointer<Pointee>, count: Int = 1) {
```

Comment:
Default parameter values need to be specified in the docs for the parameter: "The default value is `1`". Noted here and 4–5 other places below.